server/src/main/java/de/pnp/manager/network/ClientHandler.java

```diff
@@ -159,6 +161,24 @@ public StringProperty clientNameProperty() {
         return controlledCharacters;
     }
 
+    @Override
+    public Collection<String> getAccessibleInventories() {
+        return accessibleInventories;
+    }
+
+    @Override
+    public boolean hasAccessToInventory(String id) {
+        return controlledCharacters.contains(id) || accessibleInventories.contains(id);
+    }
+
```

Comment:
technically, only a character of a player has access to an inventory.
more specifically, if player A has characters X and Y and there is an inventory Z that X has access to, but Y not, only X can transfer inventories to Z, Y can not. So "hasAccessToInventory" should be character dependent.

But that is probably something for a future issue

Reply:
Agreed